### PR TITLE
Feature/update to polkadot v0 9 0

### PR DESCRIPTION
**Pull request by MRamanenkau**
@AndreiNavoichyk
Checked primitives, frame, utils, clien.
Found diffs:
1. `use sp_std::{prelude::*, rc::Rc, vec};` vs `use sp_std::{prelude::*, rc::Rc};` in primitives/npos-elections/src/phragmms.rs
2. `Config` vs `Trait` in frame/support/test/tests/pallet_with_name_trait_is_valid.rs
3. `sc-tracing = { version = "3.0.0", path = "../tracing" }` vs empty line in client/executor/Cargo.toml
4. `test-case = "2.2.1"` vs empty line in client/executor/Cargo.toml

### CODE REVIEW

File: bin/node/runtime/src/lib.rs [hunk 1]
```diff
@@ -521,14 +523,25 @@ parameter_types! {
 		.get(DispatchClass::Normal)
 		.max_extrinsic.expect("Normal extrinsics have a weight limit configured; qed")
 		.saturating_sub(BlockExecutionWeight::get());
+	// Solution can occupy 90% of normal block size
+	pub MinerMaxLength: u32 = Perbill::from_rational(9u32, 10) *
+		*RuntimeBlockLength::get()
+		.max
+		.get(DispatchClass::Normal);
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:530]: Makes it possible to an offchain worker always submit "something acceptable" to the chain. 
[The issue](https://github.com/paritytech/substrate/issues/8315)
[Original PR](https://github.com/paritytech/substrate/pull/8317)

File: bin/node/runtime/src/lib.rs [hunk 2]
```diff
@@ -689,6 +703,8 @@ impl pallet_membership::Config<pallet_membership::Instance1> for Runtime {
 	type PrimeOrigin = EnsureRootOrHalfCouncil;
 	type MembershipInitialized = TechnicalCommittee;
 	type MembershipChanged = TechnicalCommittee;
+	type MaxMembers = TechnicalMaxMembers;
+	type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:707]: Required for benchmarking pallet_membership

File: bin/node/runtime/src/lib.rs [hunk 3]
```diff
@@ -119,7 +119,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 284,
+	spec_version: 286,
```
Comment (AndreiNavoichyk) [inline bin/node/runtime/src/lib.rs:122]: Why 286 and not 285?

**Review by AndreiNavoichyk — COMMENTED** [review]
@MRamanenkau 
At what version did these changes appear? Let's fix them there.

File: bin/node/runtime/src/lib.rs (hunk 3 above)
Comment (MRamanenkau) [reply bin/node/runtime/src/lib.rs:122]: I don't know the resaon why git show such diff.
In the [0.8.30 update](https://github.com/Cerebellum-Network/pos-network-node/pull/141/files#diff-67684005af418e25ff88c2ae5b520f0c040371f1d817e03a3652e76b9485224aR122) we `spec_version: 285`

**Comment by MRamanenkau** [comment]
> @MRamanenkau At what version did these changes appear? Let's fix them there.

The diff is not significant. At the same time it might be time consuming to rebuild the artifacts and the tests (also, it would be nice to run our blockchain tests against new artifacts). I propose to fix them in the next version.

**Comment by AndreiNavoichyk** [comment]
> > @MRamanenkau At what version did these changes appear? Let's fix them there.
> 
> The diff is not significant. At the same time it might be time consuming to rebuild the artifacts and the tests (also, it would be nice to run our blockchain tests against new artifacts). I propose to fix them in the next version.

Agree. Let's fix in 0.9.3

**Review by AndreiNavoichyk — DISMISSED** [review]

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**